### DISCUSSION TITLE
Load multiple copies of a container?

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1129] [Questions & Problems]
Is there a way to load multiple copies of a container all at once instead of doing it one by one?

**Comment by msevestre** [comment]
Nothing like this. How would you define in one shot the "where" to attach the loaded container?

**Reply by moPellowe** [reply]
Do you mean "where" as in where in the diagram view? Because if I'm thinking of the tree view, I would have thought that all of the containers would end up in the container that you're loading into.

**Reply by msevestre** [reply]
Yes. In that case you would have rename each single container as name has to be unique within a parent container.

**Reply by moPellowe** [reply]
True. Although I noticed that when you load a container with a name that is already being used, then it iterates for you (e.g. Bin, Bin 1, Bin 2, etc.). Perhaps it could automatically iterate like this when loading multiple copies?
